packages/components/codemods/Well/transformWell.spec.ts

```diff
@@ -0,0 +1,186 @@
+import path from "path"
+import ts from "typescript"
+import { transformWellSource, updateFileContents } from "./transformWell"
```

Comment:
Probably worth renaming either renaming this function to be more specific about variant to color (like the other function you made), or we create a directory to keep the scope. This prevents clashing if we have further changes we want to make to Well :)